menugengold.rb

```diff
@@ -0,0 +1,40 @@
+# dinner level of project completed
+# string literal to create arrays
+descriptor = %w(sticky sloppy burning tender awesome frozen blackened roasted tepid wilted candied crusty boiled mushy pungent)
+food = %w(tuna chicken bean tofu eggplant mushroom cheese veggie pork steak bison kangaroo grasshoppper fruit egg)
+container = %w(stir-fry salad sandwich burger casserole taco pasta dumpling sushi burrito pierogi cake pie cookie smoothie)
+
+# welcome string
+puts "Welcome to Chez Elia! Would you like to decide how many items are on the menu? \n(Y or N)"
+response = gets.chomp.upcase
+
+# allow user to see the number of menu items they request, accounts for errant input
+# and the length of the selector arrays
+if response == "Y" || response == "YES"
+  puts "Please select the number of menu items you would like (up to #{descriptor.length}):"
+  menu_length =  gets.chomp
+  while menu_length != "0" && menu_length.to_i == 0 || menu_length.to_i > descriptor.length
+    puts "#{menu_length} is not a valid number. What number would you like?"
```

Comment:
This seems a little misleading for the user if you type a number where `menu_length.to_i > descriptor.length`. It could still be a valid number in the abstract, but not valid input for this program.